MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -76,11 +76,32 @@ import Foundation
         }
     }
 
+    // Retrieves the access token for the default OIDC(openid, offline_access, profile) scopes from the cache.
```

Comment:
```suggestion
    // Retrieves the access token for the currently signed in account from the cache for the provided parameters.
```
It's not always true that we will retrieve the token for the default OIDC scopes.